packages/core/src/Table/TableHeader.tsx

```diff
@@ -94,10 +94,10 @@ export const TableHeader: React.FC<TableHeaderProps> = ({
       {overlay !== undefined ? (
         <OverlayContainer>{overlay}</OverlayContainer>
       ) : (
-        React.Children.map(children, (cell, i) => {
-          return <TableHeaderCellContent key={i}>{cell}</TableHeaderCellContent>
-        })
-      )}
+          React.Children.map(children, (cell, i) => {
+            return <TableHeaderCellContent data-col={i} key={i}>{cell}</TableHeaderCellContent>
+          })
+        )}
```

Comment:
If this reformatting is because of some change in prettier, we should probably commit it separately.